Sources/PIALibrary/Account/Data/Networking/SignupRequestConfiguration.swift

```diff
@@ -0,0 +1,19 @@
+
+import Foundation
+import NWHttpConnection
+
+struct SignupRequestConfiguration: NetworkRequestConfigurationType {
+    let networkRequestModule: NetworkRequestModule = .account
+    let path: RequestAPI.Path = .signup
+    let httpMethod: NWHttpConnection.NWConnectionHTTPMethod = .post
+    let inlcudeAuthHeaders: Bool = false
+    
+    // Refreshing the auth tokens is not needed before executing the refresh API token request
+    let refreshAuthTokensIfNeeded: Bool = false
```

Comment:
This property is no longer part of `NetworkRequestConfigurationType`, we just refresh the tokens directly from the use cases that need it.

Reply:
Thank you! Will update it on the next PR